### DISCUSSION TITLE
IP Address Mapping

### DISCUSSION THREAD

**Discussion by JHarrison712** [netbox-community/netbox#13024] [Help Wanted!]
I have an ESXi server when I input the IP of the ESXi Device it couldn't automatic fill it on the other end when the cable is connected say a switch.  I would've thought it would link them in both places so say your doing a switch audit you know what IP is being used along with on the esxi server or etc. 

ESXI - https://i.imgur.com/yFLPLtS.png
Switch - https://i.imgur.com/bKzhNgX.png

**Comment by jcollie** [comment]
No, that's working as intended. If you want to know what IP addresses are being used by devices attached to a switch you have to follow the connections to the connected device.

**Reply by JHarrison712** [reply]
I gotcha, didn't know if we can make it show up in both places without duplicating IP
